sklearn/metrics/_pairwise_distances_reduction/_radius_neighborhood.pyx.tp

```diff
@@ -128,7 +128,6 @@ cdef class RadiusNeighbors{{name_suffix}}(BaseDistanceReducer{{name_suffix}}):
                 datasets_pair=DatasetsPair{{name_suffix}}.get_for(X, Y, metric, metric_kwargs),
                 radius=radius,
                 chunk_size=chunk_size,
-                metric_kwargs=metric_kwargs,
```

Comment:
If `RadiusNeighbors.compute` was a public API, then not passing in `metric_kwargs` to the generic implementation is strange. I see that `metric_kwargs` currently doe snothing in `RadiusNeighbors`, so I am okay with removing it.

Can the docstring be updated to state that `metric_kwargs` is only used when the metric is euclidean?

Reply:
Yes, this was a copy-paste mistake.